CalibMuon/DTCalibration/plugins/SealModule.cc

```diff
@@ -48,7 +50,7 @@ DEFINE_FWK_MODULE(DTTTrigWriter);
 DEFINE_FWK_MODULE(DTTTrigCorrection);
 DEFINE_FWK_MODULE(DTTTrigCorrectionFirst);
 DEFINE_FWK_MODULE(DTT0Calibration);
-//DEFINE_FWK_MODULE(DTT0CalibrationNew);
+DEFINE_FWK_MODULE(DTT0CalibrationNew);
```

Comment:
what is the plan for "new" becoming standard? Can we remove the old one?

Reply:
At the moment the "new" DTT0 calibration is mostly used but the old method is still used sometimes for crosschecks. We are working on a rewrite of the DTT0 calibration code and will replace this class in a future PR.

Reply:
@tobias-pook - please rename as V2 or some other descriptive identifier rather than "new" which is rather meaningless after a while - these replacements sometimes seem to take much longer than originally hoped..